tss-esapi/src/tcti_ldr.rs

```diff
@@ -247,6 +256,16 @@ impl FromStr for TctiNameConf {
             )?));
         }
 
+        let libtpms_pattern = Regex::new(r"^libtpms(:(.*))?$").unwrap(); //should not fail
```

Comment:
Too bad we've got parsing all these regexes here, instead of some static variable... but the same happens with others so it's :+1: